main_app/templates/main_app/helpline_numbers.html

```diff
@@ -132,108 +133,1384 @@ <h1 class="wb-stl-heading1">
-                    </td>
-                    <td style="height: 100px; width: 150px; padding-top: 35px;">
-                        <p class="wb-stl-normal"><strong><span class="wb-stl-special">011-23385368 &nbsp; &nbsp;<a href="tel:011-23385368"><i class="fa fa-phone"
+                                        aria-hidden="true"></i></a></span></span></strong></span>
+                            </p>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td style="height: 100px; padding-top: 35px;" width="275">
+                            <p class="wb-stl-normal"><span class="wb-stl-special">&#8226; &nbsp; &nbsp; National Human Right Commision</span></p>
+                        </td>
+                        <td style="height: 100px; width: 150px; padding-top: 35px;">
+                            <p class="wb-stl-normal"><strong><span class="wb-stl-special">011-23385368 &nbsp; &nbsp;<a href="tel:011-23385368"><i class="fa fa-phone"
                                     aria-hidden="true"></i></a>&nbsp; &nbsp;<br>
                                     9810298900 &nbsp; &nbsp;<a href="tel:9810298900"><i class="fa fa-phone"
                                     aria-hidden="true"></i></a></span></strong></p>
-                    </td>
-                </tr>
-                <tr>
-                    <td style="height: 100px; padding-top: 35px;" width="275">
-                        <p class="wb-stl-normal"><span class="wb-stl-special">&#8226; &nbsp; &nbsp; National Commision for Women</span></p>
-                    </td>
-                    <td style="height: 100px; width: 150px; padding-top: 35px;">
-                        <p class="wb-stl-normal"><strong><span class="wb-stl-special">011-23237166 &nbsp; &nbsp;<a href="tel:011-23237166"><i class="fa fa-phone"
+                        </td>
+                    </tr>
+                    <tr>
+                        <td style="height: 100px; padding-top: 35px;" width="275">
+                            <p class="wb-stl-normal"><span class="wb-stl-special">&#8226; &nbsp; &nbsp; National Commision for Women</span></p>
+                        </td>
+                        <td style="height: 100px; width: 150px; padding-top: 35px;">
+                            <p class="wb-stl-normal"><strong><span class="wb-stl-special">011-23237166 &nbsp; &nbsp;<a href="tel:011-23237166"><i class="fa fa-phone"
                                     aria-hidden="true"></i></a>&nbsp; &nbsp;
                                     <br>23234918 &nbsp; &nbsp;<a href="tel:23234918"><i class="fa fa-phone"
                                     aria-hidden="true"></i></a></span></strong></p>
+                        </td>
+                    </tr>
+
+                </tbody>
+            </table>
+            <br>
+            <br>
+            <br>
+        </div>
+
+        <div class="column">
```

Comment:
Just add `margin-top:9%` in the style here as `STATE WISE WOMEN HELPLINE NUMBERS` is overlapping with the table text. Then it is ready to be merged.

Reply:
@sakshikhachane sorry bt can u send me the ss of overlapping? so that i can manage accordingly.

Reply:
![image](https://user-images.githubusercontent.com/53635792/112489010-0ef4a880-8da4-11eb-8909-3b896eac063d.png)